trino/sqlalchemy/compiler.py

```diff
@@ -102,6 +112,38 @@ def limit_clause(self, select, **kw):
             text += "\nLIMIT " + self.process(select._limit_clause, **kw)
         return text
 
+    def visit_column(self, column, add_to_result_map=None, include_table=True, **kwargs):
```

Comment:
Why do we need to use visitor pattern instead of just setting `X-Trino-Catalog` or `X-Trino-Schema` header?

Reply:
Thanks for your explanation. I understand `X-Trino-Catalog` doesn't fit such use cases.